docs/users/overpass-api-guide.md

```diff
@@ -0,0 +1,318 @@
+(
+  way["leisure"="park"]({{bbox}});
+  relation["leisure"="park"]({{bbox}});
+);
+out geom;
+```
+
+**Important:** Connect the `BoundsOp` output to the `OverpassOp` bbox input. The `{{bbox}}` template will only be replaced when a valid bounding box is connected. Without a bbox connection, queries using `{{bbox}}` will fail.
+
+#### Understanding Overpass QL
+
+- **`[out:json]`**: Return results as JSON
+- **`[timeout:25]`**: Query timeout in seconds
+- **`way["leisure"="park"]`**: Ways (areas/lines) tagged as parks
+- **`relation["leisure"="park"]`**: Relations (complex geometries) tagged as parks
+- **`({{bbox}})`**: Bounding box placeholder - automatically replaced with coordinates
+- **`out geom;`**: Include full geometry in output
+
+### Step 3: Visualize with GeoJsonLayer
+
+Add a `GeoJsonLayerOp` and configure:
+
+- **data**: Connect to `OverpassOp` data output
+- **filled**: `true`
+- **getFillColor**: `#10b981` (green for parks)
+- **opacity**: `0.7`
+- **stroked**: `true`
+- **getLineColor**: `#059669` (darker green border)
+- **getLineWidth**: `2`
+
+### Step 4: Display on Map
+
+Add a `ViewerOp` and connect the `GeoJsonLayerOp` layer output.
+
+### Complete Node Graph
+
+```
+[BoundsOp] → [OverpassOp] → [GeoJsonLayerOp] → [ViewerOp]
+  point1:      query: (Overpass QL)  data: connected     layers: connected
+  [-74.05,     bbox: connected       getFillColor: #10b981
+   40.68]      endpoint: default     opacity: 0.7
+  point2:
+  [-73.90,
+   40.82]
+```
+
+## More Examples
+
+### Cafes and Restaurants
+
+Query amenities with specific tags:
+
+```
+[out:json][timeout:25];
+(
+  node["amenity"="cafe"]({{bbox}});
+  node["amenity"="restaurant"]({{bbox}});
+);
+out;
+```
+
+Then visualize as points with `GeoJsonLayerOp`:
+
+- **pointType**: `circle`
+- **getPointRadius**: `8`
+- **pointRadiusUnits**: `pixels`
+- **getFillColor**: Use accessor to color by type:
+  ```javascript
+  d => d.properties.amenity === 'cafe' ? [139, 92, 246] : [234, 88, 12]
+  ```
+
+### Bike Lanes
+
+Query cycling infrastructure:
+
+```
+[out:json][timeout:25];
+(
+  way["highway"="cycleway"]({{bbox}});
+  way["cycleway"]({{bbox}});
+  way["bicycle"="designated"]({{bbox}});
+);
+out geom;
+```
+
+Visualize with:
+
+- **stroked**: `true`
+- **filled**: `false`
+- **getLineColor**: `#2563eb`
+- **getLineWidth**: `3`
+- **lineWidthUnits**: `pixels`
+
+### Historic Buildings
+
+Query buildings with historic significance:
+
+```
+[out:json][timeout:25];
+(
+  way["historic"="yes"]({{bbox}});
+  way["building"]["historic"]({{bbox}});
+  relation["historic"]({{bbox}});
+);
+out geom;
+```
+
+### Trees in a Park
+
+Query natural features within a specific area (without using `{{bbox}}` template):
+
+```
+[out:json][timeout:25];
+(
+  node["natural"="tree"](40.7829,-73.9654,40.7850,-73.9630);
+);
+out;
+```
+
+Note: This uses a fixed bbox instead of the `{{bbox}}` template for precise areas. Simply don't connect a BoundsOp to the bbox input.
+
+## Advanced Queries
```

Comment:
A few full examples are great. I think this "advanced" section gets into material that ought to come from another source